### ISSUE TITLE
CONST not evaluating properly when using a literal

### ISSUE THREAD

**Issue by ghost** [QB64-Phoenix-Edition/QB64pe#224]
When using the below code with the latest source (clone and run setup_win.bat), this CONST expression does not work. It did work in 1.5. If I replace the `&HFFF` literal with a new constant that is set to that value then the IDE stops complaining.
```
    Const STANDARD_RIGHTS_REQUIRED = &H000F0000
    Const SYNCHRONIZE = &H00100000
    Const PROCESS_ALL_ACCESS = STANDARD_RIGHTS_REQUIRED Or SYNCHRONIZE Or &HFFF
```

**Comment by mkilgore** [comment]
It turns out it's quite simple, but also probably a mess to fix. Basically in `PreParse()` in the "PP_TypeMod" step all type suffixes are replaced with a "C_*" equivalents which is used to (I guess) make handling them easier later by turning them into a kind of function. The big issue is that the logic can't tell that the `&` is not a type suffix, so it gets replaced with `C_LO` which obviously then breaks everything after.

The single `&H00100000` cases work because while it bombs out of the `Evaluate_Expression$` logic, the particular way they bombs out allows it to get handled by the `fixoperationorder$` and `evaluateconst$` logic which can handle that expression and produces the correct value. When the expression is after an operator like `Or` then the resulting string bombs out of the `Evaluate_Expression$` logic with an error that stops the `CONST` evaluations and never calls `evaluateconst$`.

Sadly my const test suite is missing cases for `&H` and friends, or else this probably would have been noticed. Cases can be easily added at least.

**Comment by mkilgore** [comment]
Fixed with #397
